Description: watch.8 --chgexit only on visible changes
 watch only exits on changes that are visible. This is a by-product
 of how changes are detected which is in the ncurses idea of windows.
 .    
 The limits of change detection is also why multicolours can't be
 implemented, so a large re-work of how the output is stored and
 displayed may fix both.
Origin: upstream, https://gitlab.com/procps-ng/procps/-/commit/f2ea334fe3f618c3a08e64f5ef0b5ef8e62add1f
Bug-Debian: https://bugs.debian.org/729569
Applied-Upstream: 4.0.5
Last-Update: 2025-04-14
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/man/watch.1
+++ b/man/watch.1
@@ -1,5 +1,5 @@
 .\"
-.\" Copyright (c) 2009-2023 Craig Small <csmall@dropbear.xyz>
+.\" Copyright (c) 2009-2024 Craig Small <csmall@dropbear.xyz>
 .\" Copyright (c) 2018-2023 Jim Warner <james.warner@comcast.net>
 .\" Copyright (c) 2011-2012 Sami Kerola <kerolasa@iki.fi>
 .\" Copyright (c) 2003      Albert Cahalan
@@ -10,7 +10,7 @@
 .\" (at your option) any later version.
 .\"
 .\"
-.TH WATCH 1 "2023-01-17" "procps-ng" "User Commands"
+.TH WATCH 1 "2024-07-19" "procps-ng" "User Commands"
 .SH NAME
 watch \- execute a program periodically, showing output fullscreen
 .SH SYNOPSIS
@@ -44,9 +44,9 @@
 Freeze updates on command error, and exit after a key press.
 .TP
 \fB\-g\fR, \fB\-\-chgexit\fR
-Exit when the output of
-.I command
-changes.
+Exit when the visible output of \fIcommand\fR changes. Changes that are off
+the screen due to small screen size or large output will not cause \fBwatch\fR
+to exit.
 .TP
 \fB\-n\fR, \fB\-\-interval\fR \fIseconds\fR
 Specify update interval.  The command will not allow quicker than 0.1 second
